src/IO.agda

```diff
@@ -27,6 +27,7 @@ private
 -- Re-exporting the basic type and functions
 
 open import IO.Base public
+open import IO.Handle public
```

Comment:
I feel the fact that `IO` now exports a load new operations should be reflected in the CHANGELOG?

Reply:
Done